src/Polly.Core.Specs/Polly.Core.Specs.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's the reason for using the "Specs" postfix? 

nit: How about just use the standard `Polly.Core.Tests` name?  I must admit I have not seen this `.Specs` convention anywhere and it got me confused at first.

@martincostello @joelhulen

Reply:
@joelhulen your opinion here? 

Shall we stick with "Specs" for new tests?

Reply:
@martintmk Specs is legacy from around 2013 or so. I'm OK with updating this to something else like `Tests`.

Reply:
Cool, I have renamed these to `Polly.Core.Tests` then.